File: result/translate/chatgpt/python2java/java/NUMBER_SUBSEQUENCES_AB_STRING_REPEATED_K_TIMES.java
public static int f_gold(String s, int K) {
    int n = s.length();
    int c1 = 0;
    int c2 = 0;
    int C = 0;
    for (int i = 0; i < n; i++) {
        if (s.charAt(i) == 'a') {
            c1 += 1;
        }
        if (s.charAt(i) == 'b') {
            c2 += 1;
            C += c1;
        }
    }
    return C * K + (int) (K * (K - 1) / 2) * c1 * c2;
}

File: result/translate/starchat/java2python/python/NUMBER_SUBSEQUENCES_AB_STRING_REPEATED_K_TIMES.py
def f_gold(s, K):
        n = len(s)
        C = 0
        c1 = 0
        c2 = 0
        for i in range(n):
            if s[i] == 'a':
                c1 += 1
            if s[i] == 'b':
                c2 += 1
                C += c1
        return C * K + (K * (K - 1) // 2) * c1 * c2